lib/assets/javascripts/firehose/web_socket.js.coffee

```diff
@@ -111,7 +111,7 @@ class Firehose.MultiplexedWebSocket extends Firehose.WebSocket
     @_sendMessage
       multiplex_subscribe:
         channel: channel
-        message_sequence: opts.last_sequence
+        last_message_sequence: opts.last_sequence
```

Comment:
@bakkdoor Are you confident that the server is still backwards compatible with this change? We want to avoid having to deploy client and server changes at the same time. Some clients can be very slow to update (such as mobile apps). And even web apps would require a page refresh, which is nice to avoid.